### PR TITLE
Add tooltip for copying address to clipboard in dashboard

### PR DESCRIPTION
**Pull request by hskang9**
# What I did
Add tooltip on mouseenter of the address.

Screen Shot 2019-05-12 at 12 09 31 AM

# Cute animal picture

image

### CODE REVIEW

File: src/ts/components/dashboard/Dashboard.tsx [hunk 1]
```diff
@@ -20,9 +20,13 @@ import { KeyringPair$Json } from '@polkadot/keyring/types'
 import Balance from '../account/Balance'
 import Identicon from 'polkadot-identicon'
 import { saveSettings } from '../../background/store/settings'
+import { Popup } from 'semantic-ui-react'
```
Comment (ttaoS) [inline src/ts/components/dashboard/Dashboard.tsx]: It is a duplicate import

File: src/ts/components/dashboard/Dashboard.tsx [hunk 2]
```diff
@@ -220,9 +224,11 @@ class Dashboard extends React.Component<IDashboardProps, IDashboardState> {
           </Dropdown>
         </Section>
         <Section>
+          <Tooltip title='Copy to clipboard' position='bottom' trigger='mouseenter' arrow={true}>
```
Comment (ttaoS) [inline src/ts/components/dashboard/Dashboard.tsx]: it looks good i will remove my css which is also used for copy tooltip

File: src/ts/components/dashboard/Dashboard.tsx (hunk 1 above)
Comment (ttaoS) [reply src/ts/components/dashboard/Dashboard.tsx]: i will push a fix to this branch to fix it along with my css change

Comment (hskang9) [reply src/ts/components/dashboard/Dashboard.tsx]: 👍

**Comment by ttaoS** [comment]
Only issue i can see in this pr is when you mouse over it, the screen is actually moved and a scroll bar is displayed.

**Comment by hskang9** [comment]
ok I will try to look into that.

File: src/ts/components/dashboard/Dashboard.tsx (hunk 2 above)
Comment (fyang1024) [inline src/ts/components/dashboard/Dashboard.tsx]: One minor thing @hskang9 can you please define 'Copy to clipboard' in `_locales/en/messages.json` file and use i18n service to reference that?

Comment (hskang9) [reply src/ts/components/dashboard/Dashboard.tsx]: okay will do that.

Comment (hskang9) [reply src/ts/components/dashboard/Dashboard.tsx]: done.

**Pull request merged**